### ISSUE TITLE
Profilng breakdown of camera -> display latency

### ISSUE THREAD

**Issue by jleibs** [rerun-io/rerun#6185]
I did a very adhoc test to measure closed-loop latency using a camera to capture the screen, and presenting the captured image along side the frame  comparing to cv.imshow as a reference.

Can be reproduced using the following gist:
https://gist.github.com/jleibs/08c65d6195ea5dad79e8974201258f62

My tests were on with a 1600x900 camera running at 60fps 
We can see a total round-trip phase-lag for Rerun of 9-10 frames (160ms), and for `cv.imshow` of about 8 frames.

1-frame discprency between Rerun and Imshow
![image](https://github.com/rerun-io/rerun/assets/3312232/c322810e-a1da-454b-9bea-2b71527d9bec)

2-frame disprepency between Rerun and Imshow
![image](https://github.com/rerun-io/rerun/assets/3312232/96696b08-f8ff-43bc-9f7a-f9012f5fe63d)

This 1-2 frame lag seems inline with the Rerun-measured latency of 13ms + the app/render latency of 4ms + the additional random phase offset between the display clock and the camera clock.

**Comment by jleibs** [comment]
It would be good to run a similar comparison using a Rust or C++ app with a more efficient camera -> app pipeline.

**Comment by emilk** [comment]
The default value of `RERUN_FLUSH_TICK_SECS` is also 8ms: https://www.rerun.io/docs/reference/sdk-micro-batching

**Comment by jleibs** [comment]
> The default value of `RERUN_FLUSH_TICK_SECS` is also 8ms

Good point. I don't think that should be coming into play here since `RERUN_FLUSH_NUM_BYTES` is 1MB and each image is over 4MB
